common/changes/@microsoft/rush/chao-auto-install-peers_2023-10-03-20-48.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add a new setting `autoInstallPeers` in pnpm-config.json",
```

Comment:
Since we've decided to change the default, the notice should be more explicit:
```
(IMPORTANT) Add a new setting `autoInstallPeers` in pnpm-config.json; be aware that Rush changes PNPM's default if you are using PNPM 8 or newer
```